spec/requests/nominations/nominate_induction_coordinator_spec.rb

```diff
@@ -45,13 +45,46 @@
       let(:token) { nomination_email.token }
 
       it "redirects to link-expired" do
-        get "/nominations/new?token=#{token}"
+        get "/nominations/start?token=#{token}"
 
         expect(response).to redirect_to("/nominations/link-expired?school_id=#{nomination_email.school.id}")
       end
     end
   end
 
+  describe "GET /nominations/new" do
+    let(:now) { rand(-100..100).hours.ago }
```

Comment:
Why? Daylight savings?

Reply:
This is to decouple the test from the actual time to avoid potential time-related issues (both positive and negatives). I normally like to introduce a bit of randomness into the tests as it forces me to write the tests in a more resilient way. Additionally, they do sometimes catch some edge cases, which is an extra bonus.

Reply:
Fine by me, just like hearing rationale to learn new things 🙂